docs/design/fl-parallelism/fl-alive-tracking.puml

```diff
@@ -11,10 +11,6 @@ end box
 participant "FL-router" as fl_router
```

Comment:
nit: as for me I'd change flr and fls on a diagram to read it from left to right

Reply:
done